### ISSUE TITLE
`-d` flag only finds *.yaml files and not *.yml

### ISSUE THREAD

**Issue by l3rl4n** [instrumenta/kubeval#178]
Kubeval skips any `*.yml` files when auto-detecting recursively through specified directories. 

```bash
> tree
.
├── find_me.yml
└── test_deployment.yaml

0 directories, 2 files

> kubeval -d .
The file test_deployment.yaml contains a valid Deployment

>
```

**Comment by l3rl4n** [comment]
https://github.com/instrumenta/kubeval/pull/177

**Comment by garethr** [comment]
Fixed in #177
